yt/yql/tests/agent/test_simple.py

```diff
@@ -228,6 +228,13 @@ def test_exotic_types(self, query_tracker, yql_agent):
                "Yson": [7, 'str', -3.14]
                }])
 
+    @authors("a-romanov")
+    def test_double_optional(self, query_tracker, yql_agent):
```

Comment:
Could you please add a more complex test with combination of inner maps and lists, like `[[[null],[null]],[[null],[null]]]`

To test logic with OptionalLevels and Depth